### PR TITLE
Added new rules and directories. Changed redirect to github raw format

### PR DESCRIPTION
**Pull request by ritamargherita**

### CODE REVIEW

File: odissei/np/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Nanopublication
+===============
+
```
Comment (dgarijo) [inline odissei/np/README.md:3]: What is the purpose of this file? It does not really add any information, can it be deleted?

**Comment by dgarijo** [comment]
@ritamargherita,
I think that if you are going to be managing the root w3id, you don't need to add a readme file with you as a maintainer for each sub folder. We use the readme in the main folder to know who is the maintainer of the repo. Please have a look at my comment about the empty file and once addressed I will merge

File: odissei/np/README.md (hunk 1 above)
Comment (ritamargherita) [reply odissei/np/README.md]: I have deleted the README files as recommended

**Comment by ritamargherita** [comment]
Removed README files

**Comment by dgarijo** [comment]
@ritamargherita,
looking at your redirects in detail, I think that the np redirect should be in the subfolder, because the `np` folder is for that redirection. In any case, I promised I would merge, so here it goes.

**Pull request merged**